src/main/java/uk/gov/hmcts/reform/sscs/service/AppInsightsService.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.hmcts.reform.sscs.service;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectWriter;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.sscs.model.AppInsights;
+
+@Slf4j
+@Service
+public class AppInsightsService {
+
+    public void sendAppInsights(AppInsights appInsights) throws JsonProcessingException {
+        log.info(appInsightsToJson(appInsights));
```

Comment:
Is this just a placeholder?

Reply:
No, this is the full implementation. It will take the object, stringify it and log it which will also post to app insights.